src/Scene.h

```diff
@@ -46,6 +48,8 @@ class CScene
 	void add(const ptr_camera_t pCamera)
 	{
 		// --- PUT YOUR CODE HERE ---
+		m_vpCameras.push_back(pCamera);
+		m_activeCamera = m_activeCamera + 1;
```

Comment:
This is valid approach, but only in case, when you initialize m_activeCamera to be equal to -1 on construction. This also requires to have m_activeCamera variable of type int, not size_t.
How it is implemented now, it shows an error on execution.